frontend/src/lib/k8s/service.ts

```diff
@@ -22,6 +42,20 @@ class Service extends makeKubeObject<KubeService>('service') {
   get spec() {
     return this.jsonData!.spec;
   }
+
+  get status() {
+    return this.jsonData!.status;
+  }
+
+  getExternalAddresses() {
+    return _.uniq(
+      (
+        this.status?.loadBalancer?.ingress?.map(
+          (ingress: KubeLoadBalancerIngress) => ingress.ip || ingress.hostname
```

Comment:
I wonder why the ip address takes precedence over the hostname here?

Reply:
Right. I think this is a mistake. I will change it